### PR TITLE
Added option for own debug node commandline arguments to Remote Debugger

### PR DESCRIPTION
**Pull request by vikwin**
Added an option that allows the user to specify further commandline arguments when using the remote debug feature. That can be useful for example if the user want's to include other binary folders using the '-pa xyz' argument.

### CODE REVIEW

File: src/org/intellij/erlang/debugger/remote/ErlangRemoteDebugRunningState.java [hunk 1]
```diff
@@ -76,13 +78,21 @@ public ConsoleView createConsoleView(Executor executor) {
...
+      String qualifiedName = getNodeName() + "@" + (host == null ? getDefaultHost() : host);
+      result = new ArrayList<String>(ContainerUtil.list("-name", qualifiedName));
     }
 
-    String host = StringUtil.nullize(myConfiguration.getHost(), true);
-    String qualifiedName = getNodeName() + "@" + (host == null ? getDefaultHost() : host);
-    return ContainerUtil.list("-name", qualifiedName);
+    List<String> debugNodeArgs = Arrays.asList(myConfiguration.getDebugNodeArgs().split(" "));
+    result.addAll(debugNodeArgs);
+    return result;
```
Comment (deadok22) [inline src/org/intellij/erlang/debugger/remote/ErlangRemoteDebugRunningState.java]: Use ```ContainerUtil.concat(List, List)```

File: src/org/intellij/erlang/debugger/remote/ErlangRemoteDebugRunningState.java [hunk 2]
```diff
@@ -76,13 +78,21 @@ public ConsoleView createConsoleView(Executor executor) {
...
+    } else {
+      String host = StringUtil.nullize(myConfiguration.getHost(), true);
+      String qualifiedName = getNodeName() + "@" + (host == null ? getDefaultHost() : host);
+      result = new ArrayList<String>(ContainerUtil.list("-name", qualifiedName));
     }
 
-    String host = StringUtil.nullize(myConfiguration.getHost(), true);
-    String qualifiedName = getNodeName() + "@" + (host == null ? getDefaultHost() : host);
-    return ContainerUtil.list("-name", qualifiedName);
+    List<String> debugNodeArgs = Arrays.asList(myConfiguration.getDebugNodeArgs().split(" "));
```
Comment (deadok22) [inline src/org/intellij/erlang/debugger/remote/ErlangRemoteDebugRunningState.java]: Use com.intellij.util.execution.ParametersListUtil#parse(java.lang.String)

**Comment by vikwin** [comment]
@deadok22 
I changed the code to match your suggestions.

**Comment by deadok22** [comment]
Ok, I've ran the changes and they work fine.

There are two things left to do:
1) Reassign mnemonic to something different - 'a' conflicts with 'Apply' in run configuration editing dialog
2) Format your changes (*Code* | *Reformat Code* while having your changes selected)

Once this is done, it's ready to be merged.

**Comment by vikwin** [comment]
@deadok22 
Ready to be merged :+1:

**Comment by deadok22** [comment]
Thank you for contribution. 

I had to change the mnemonic again - the one you reassigned it to had conflicts again. Also, I did some cleanup - you may want to take a look at it.

Cheers!

**Comment by vikwin** [comment]
@deadok22 
Is there any chance you could put up a build including the current changes? I have problems with this after importing the project in IntelliJ, as it does not recognize the subcomponents as plugin projects.

**Comment by deadok22** [comment]
You can grab a build from [here](https://teamcity.jetbrains.com/project.html?projectId=IntellijIdeaPlugins_Erlang&tab=projectOverview). Also, you don't need IntelliJ to build the plugin - there is a gradle target for building a plugin zip.

**Pull request merged**